<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1304</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
    background-color : #F5F5F5;

}

QPushButton 
{
    background: transparent;
	border:2px groove #1ECD97 ;
	color:#1ECD97;
	
	font-size:19px;
	font-weight:bold;
	padding:15px 17px;
	text-decoration:none;
font-family: 'Montserrat', sans-serif;
}

QPushButton:hover 
{
	background-color:#1ECD97 ;
    color: white;
}

QPushButton:active 
{
	position:relative;
	top:1px;
}
QPushButton#pushButton_2
{
    background: transparent;
	border:2px groove #eb675e ;
	color:#1ECD97;
	
	font-size:19px;
	font-weight:bold;
	padding:15px 17px;
	text-decoration:none;
font-family: 'Montserrat', sans-serif;
}
QPushButton#pushButton_2:hover 
{
	background-color:#eb675e ;
    color: white;
}

QPlainTextEdit
{
border:2px groove;
}

QLabel
{
border:1px solid;
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="labelOriginal">
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="labelProcessed">
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>90</height>
       </size>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>90</height>
       </size>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="3">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>180</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
